HomeScreenLib/src/main/res/layout/activity_home.xml

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <variable
+            name="dashboardViewmodel"
+            type="com.example.homescreenlib.viewmodel.DashboardViewModel" />
+        <!--<variable
+            name="adapter"
+            type="com.example.homescreenlib.ui.HomeAdapter" />-->
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="#dfe6f6"
```

Comment:
use resource file to get colours and other themes related stuffs

Reply:
Completed please check the latest commit